README.md

```diff
@@ -22,24 +22,103 @@ We recognise the following major repositories and organisations may create `<rep
 ### Package list schema
 
 Package lists are formatted as JSON.
-Each file is a list of package entries with the following fields.
+The schema for major repositories is formally described in [`schema.json`](./schema.json).
+This is a [JSON Schema](https://json-schema.org/overview/what-is-jsonschema) document using the 2020-12 draft.
+
+In brief, each file is a list of package entries with the following fields.
 
 - `package_name` (text, case sensitive, uniquely defines a package for a repository)
-- `version` (text, [PEP440 style](https://peps.python.org/pep-0440/#version-specifiers))
+- `version` (text, comparison operator plus [PEP440 style](https://peps.python.org/pep-0440/#version-specifiers) version number)
 - `approval_date` (ISO 8601)
 - `revoke_date` (ISO 8601 or `null` if not revoked)
 
 For example,
 
 ```JSON
 [
-    {
-        "package_name": "numpy",
-        "version": "*",
-        "approval_date": "2023-09-29",
-        "revoke_date": null
-    }
+  {
+    "package_name": "numpy",
+    "version": "== 1.2.3",
+    "approval_date": "2023-09-29",
+    "revoke_date": null
+  }
 ]
 ```
 
 Additionally for packages not in a major repository, _i.e._ those in `other.json`, a `url` field is added to uniquely identify the package and give its source.
+The other schema is described in [`schema_other.json`](schema_other.json).
```

Comment:
```suggestion
The `other` schema is described in [`schema_other.json`](schema_other.json).
```

Reply:
I wouldn't use the code style there as other isn't a variable, file name, _etc._.

Do you think the language is unclear? Maybe quoting it would make more sense?

Reply:
Maybe quotes. It's because it's another named thing - it's a schema for Other, rather than schema other than the one the that has already been defined. Perhaps actually just Other - I'd write PyPi schema rather than "PyPi" schema, after all.